.github/workflows/build_artifacts_on_PR.yml

```diff
@@ -0,0 +1,25 @@
+name: Build artifact on PR
```

Comment:
By PR I meant Pull Request - I wanted to create artifact on every pull request so that I could download app with changes that are requested to merge to base branch, install app and check these changes

Reply:
In this case please change to `Build artifact on Pull Request`

Reply:
Fix is in https://github.com/wttech/cogboard-mobile/pull/158